models/cut_model.py

```diff
@@ -237,19 +268,29 @@ def __init__(self, opt, rank):
         losses_D = ["D_tot"]
         if opt.alg_cut_nce_idt and self.isTrain:
             losses_G += ["G_NCE_Y"]
+        if opt.model_multimodal and self.isTrain:
+            losses_E = ["G_z"]
+            losses_G += ["G_z"]
```

Comment:
Maybe we could rename `"G_z"` into `"G_random_z"` to make it easier to understand